WordPress/Classes/Utility/Universal Links/Routes+Reader.swift

```diff
@@ -106,6 +109,10 @@ extension ReaderRoute: NavigationAction {
             if let (blogID, postID) = blogAndPostID(from: values) {
                 coordinator.showPost(with: postID, for: blogID, isFeed: false)
             }
+        case .wpcomPost:
+            if let urlString = values["matched-route-url"], let url = URL(string: urlString) {
```

Comment:
I wonder if we should also double check that post_year, post_month, post_day are all numbers? 🤔 Otherwise, we have no guarantee that this is definitely the type of URL we want, other than it has 4 path components.